fuzz/fuzz_targets/deserialize_prefilled_transaction.rs

```diff
@@ -0,0 +1,61 @@
+extern crate bitcoin;
+
+fn do_test(data: &[u8]) {
+    // We already fuzz Transactions in `./deserialize_transaction.rs`.
+    let tx_result: Result<bitcoin::util::bip152::PrefilledTransaction, _> = bitcoin::consensus::encode::deserialize(data);
+
+    match tx_result {
+        Err(_) => {},
+        Ok(mut tx) => {
+            let ser = bitcoin::consensus::encode::serialize(&tx);
+            assert_eq!(&ser[..], data);
+        }
+    }
+}
+
+#[cfg(feature = "afl")]
+#[macro_use] extern crate afl;
+#[cfg(feature = "afl")]
+fn main() {
+    fuzz!(|data| {
+        do_test(&data);
+    });
+}
+
+#[cfg(feature = "honggfuzz")]
+#[macro_use] extern crate honggfuzz;
+#[cfg(feature = "honggfuzz")]
+fn main() {
+    loop {
+        fuzz!(|data| {
+            do_test(data);
+        });
+    }
+}
```

Comment:
It looks like these could be a macro, I think they are repeated in more places.

Reply:
AFAIK these templates have not changed in many years and it's easy enough to just copy/paste existing fuzztests rather than adding more indirection.

But sure, I wouldn't be opposed to macro-izing all the existing fuzztests, especially if we bring more crates into this repo.